website/content/commands/connect/envoy.mdx

```diff
@@ -75,6 +75,10 @@ Usage: `consul connect envoy [options] [-- pass-through options]`
   In cases where either assumption is violated this flag will prevent the
   command attempting to resolve config from the local agent.
 
+- `envoy_hcp_metrics_bind_socket_dir` - Specifies the directory of a unix socket 
```

Comment:
Would `Specifies the directory where a unix socket is created` be better here?

Reply:
What creates the directory? The passive voice sounds as if either Envoy or Consul creates the directory. If that's true, then we should say something like "Specifies the directory where <Envoy/Consul> creates a unix socket. Envoy sends metrics to the socket so that HCP collectors can connect collect them."

Reply:
Fixed that's much better, thank you.